src/qt/bitcoingui.cpp

```diff
@@ -701,6 +708,22 @@ void BitcoinGUI::openClicked()
     }
 }
 
+void BitcoinGUI::openWalletClicked()
+{
+    QString fileName = QFileDialog::getExistingDirectory(this, tr("Open Wallet"),
+                                                QString(),
+                                                QFileDialog::ShowDirsOnly
+                                                | QFileDialog::DontResolveSymlinks);
+    if (fileName.isEmpty()) return;
+
+    std::vector<std::string> errors;
+    m_node.loadWallet(QDir::toNativeSeparators(fileName).toStdString(), errors);
```

Comment:
@jnewbery @laanwj ATM the wallet is loaded in the UI thread, should I refactor to load it in the background?

Reply:
I think that's a good future enhancement. Not necessary in the initial version IMO.